src/screens/Wallet.js

```diff
@@ -118,6 +126,10 @@ class Wallet extends React.Component {
     const promise = tokens.unregisterToken(this.props.selectedToken);
     promise.then(() => {
       $('#unregisterModal').modal('hide');
+      if (hathorLib.wallet.isHardwareWallet()) {
```

Comment:
I understand the idea, but wouldn't it be better to clean the storage when starting a new software wallet? This would clean all token signatures instead of only one.

Reply:
I don't think I like this. Imagine if I have some tokens and I decide to split in two wallets, one hardware with bigger amount and one software wallet with smaller amount of each token just for daily use. So every time I start the software wallet I will lose the ledger signatures and if I start the hardware wallet I need to sign all of them again?

Reply:
If you start the software wallet you cannot start the hardware wallet unless you reset everything.

Reply:
But you reset the custom tokens saved in storage? 🤔 
I thought we kept those, even when resetting the wallet

Reply:
You removed this part of code but now when I unregister a token, then I keep the signature in storage?